karma.conf.js

```diff
@@ -221,6 +221,7 @@ module.exports = (config) => {
       'node_modules/eme-encryption-scheme-polyfill/dist/eme-encryption-scheme-polyfill.js',
 
       // load closure base, the deps tree, and the uncompiled library
+      'test/test/closure-boot.js',
```

Comment:
seems like this causes karma's debug.html to stop working.
For whatever reason, if I try to run the debug file via something like `python3 build/test.py --uncompiled --no-build --quick --auto-watch --no-browsers`, it fails to load `node_modules/google-closure-library/closure/goog/deps.js` as well as errors out because it never ends up setting up `var shaka = {}`.

Not really sure how to fix it but thought it's worth mentioning it. Maybe this file could only be configured on chromecast?

Reply:
I see the problem, and it's more general.  It has to do with the order in which scripts load.  I'll work on a fix.

What seems to happen now is that Karma inserts shaka-player.uncompiled.js directly, which uses Closure to load the individual source files of Shaka, not with createElement attached to head instead of document.write.  This allows some subsequent scripts inserted by Karma to execute before those createElement scripts load, which is what causes the problem.

In normal testing, Karma also loads scripts with createElement, so it works.  Since flattening the environment, it's only debug.html that sticks `<script>` tags directly into HTML.

Since we're on a fork of the now-dead-upstream Karma, I may be able to fix it there.

Reply:
I was wrong.  Easier to fix in closure-boot.js, since the correct behavior depends on context.

https://github.com/shaka-project/shaka-player/pull/6730